index.html

```diff
@@ -1090,27 +1073,18 @@ <h4>Proof Configuration (eddsa-jcs-2022)</h4>
 Let <var>proofConfig</var> be a clone of the <var>options</var> object.
             </li>
             <li>
-If <var>options</var>.<var>type</var> is not set to `DataIntegrityProof` and
+If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` and/or
```

Comment:
Maybe this should just read "or"?

```suggestion
If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` or
```

Reply:
@TallTed your opinion? I think this was your wording, but I can't remember.

Reply:
I prefer the accurate conjunction be used, which could be `or`, `and`, or `and/or`. I may not have always understood which was intended, so some of these conditionals may not now be correct.

* `And` (almost obviously) requires both conditions be met for the `then`.

* `Or` is (too often) treated as an `exclusive or`, meaning that _if either but not both conditions are met_, the `then` is active, but _if both conditions are met_, the `then` is _not_ active.

* `And/or` is usually read correctly to mean _if either or both conditions are met_, the `then` is active.

It might be better to change all of these phrasings to use the `either`, `both`, or `either or both` (possibly plus `neither`) conditionals to make clear the intended interpretation.